optimizely/decision/composite_service.go

```diff
@@ -69,6 +69,9 @@ func (s CompositeService) GetFeatureDecision(featureDecisionContext FeatureDecis
 				"source":          featureDecision.Source,
 			},
 		}
+		if userContext.Attributes == nil {
```

Comment:
I think nil shouldn't be an issue for us because we don't ever attempt to write to it from the SDK. It is a read-only value and we should treat it as such. No need to initialize to an empty map. @pawels-optimizely ^^

Reply:
FSC expect's attributes to be empty dict in case of null, il just move this check on the test-app side then.